src/main/java/duke/DeleteCommand.java

```diff
@@ -0,0 +1,30 @@
+package duke;
+import java.io.IOException;
```

Comment:
```suggestion
package duke;

import java.io.IOException;
```

Again just nitpicking here, but I feel that there should be a line break between the package statement and import statement.